cves/cve_2021_41773.py

```diff
@@ -48,41 +47,79 @@
 FIRST_AFFECTED_VERSION = '2.4.49'
 SECOND_AFFECTED_VERSION = '2.4.50'
 REMEDIATION = 'Upgrade Apache version to 2.4.51 or higher.'
-MITIGATION_1 = 'Change the filesystem permissions in the <Directory /> field in the configuration file from ' \
-               '"Require all granted" to "Require all denied"'
-MITIGATION_2 = f'{MITIGATION_1}\nAlso disable the cgi_module\nOn RedHat, Fedora, CentOS and other rpm based ' \
-               f'distributions:\nmv /etc/httpd/conf.modules.d/XX-cgi.conf /etc/httpd/conf.modules.d/XX-cgi.conf.disable' \
-               f'\nOn Debian, Ubuntu and other Debian derivatives:\na2dismod cgi'
-
-
-def filesystem_directory_configuration(debug, container_name):
-    """This function checks if the filesystem directory is configured to 'Require all granted' or 'Require all
-    denied'"""
+CHANGE_PERMISSIONS = 'Change the permissions from "Require all granted" to "Require all denied" in the relevant ' \
+                     'directories'
+RESTART = 'After making the changes, remember to restart the server for the modifications to take effect'
+MITIGATION_1 = f'{CHANGE_PERMISSIONS}\n{RESTART}'
+MITIGATION_2 = f'{CHANGE_PERMISSIONS}\nDisable the cgi_module and cgid_module by adding the "#" at the beginning of ' \
+               f'the cgi_module and cgid_module lines in the configuration file\n{RESTART}'
+
+
+def parse_directory_name(directory):
+    """This function returns the directory after removing unnecessary characters."""
+    if '"' in directory:
```

Comment:
Why do we need these three If's? Why not try something like: 
```
        directory = directory.replace('"', '').replace("'", '').replace(' ', '')
```
If these characters don't exist they simply won't be replaced